minutes/minutes-2017-11-13.md

```diff
@@ -0,0 +1,150 @@
+# Software Carpentry Mentoring Subcommittee Meeting
+
+__Date:__ 2017-11-13
+
+__Time:__ 22:00 UTC
+
+## Co-Chairs
+
+- Jamie Hadwin (1st)
+- Marian Schmidt (2nd)
+
+## Attending
+
+- Jamie Hadwin (1st & 2nd)
+- Marian Schmidt (2nd)
+- Kari L. Jordan (2nd)
+- Mark Laufersweiler (2nd)
+- Raniere Silva (2nd)
+- Juan Steyn (1st)
+- Jonah Duckles (2nd)
+
+## Minutes
+
+### Introductions
+
+### Recap from last meeting
+
+#### Training or Steering Committees updates from last meeting
+
+- __Training__ (Erin):
+  - Trainers: Seventeen new Trainers have been invited to join the Trainer group and will be starting the training program soon. 
+  - Maintainers: New Maintainers will be coming on board for many of the SWC/DC lessons in the next few months. The Mentoring committee may see some impacts on PR response times.
+  - Lessons: The DC Genomics lessons are scheduled to be published next week (Nov 15th). There have been a lot of reorganization and structural changes. Expect to see more people teaching these coming through discussions. 
+  - Applications were open for new trainers
+
+- __Steering__: No steering committee update this month.
+- __Mentoring Groups__ (Kari): https://github.com/carpentries/mentoring/tree/master/mentoring-program
```

Comment:
broken link, should probably be https://github.com/carpentries/mentoring/blob/master/mentoring-groups/program-outline.md